doc/man3/BN_rand.pod

```diff
@@ -37,7 +43,13 @@ If B<bits> is 1 then B<top> cannot also be B<BN_RAND_FLG_TOPTWO>.
 BN_rand_range() generates a cryptographically strong pseudo-random
 number B<rnd> in the range 0 E<lt>= B<rnd> E<lt> B<range>.
 
-The PRNG must be seeded prior to calling BN_rand() or BN_rand_range().
+BN_priv_rand() and BN_priv_rand_range() have, respectively, the same
```

Comment:
Maybe move ",respectively" to the end?  (that saves a comma.)

Reply:
@mspncp I'll fix it right away, thanks for the feedback!

Reply:
@mspncp done!